pyproject.toml

```diff
@@ -4,11 +4,14 @@ dynamic = ["version"]
 name = "pip"
 description = "The PyPA recommended tool for installing Python packages."
 readme = "README.rst"
-license = {text = "MIT"}
+license = "MIT"
+license-files = [
+  "AUTHORS.txt", "LICENSE.txt",
+  "src/pip/_vendor/**/*LICENSE*", "src/pip/_vendor/**/*COPYING*",
```

Comment:
While I'm being annoying, I'll also ask for these to be changed into one-per-line to align with how we're formatting Python files.

Reply:
Heh, I'm a pedant myself. 🙂 Updated.